impl/src/utils.rs

```diff
@@ -1385,6 +1397,31 @@ pub(crate) mod skip {
     /// ```
     pub(crate) struct Attribute(&'static str);
 
+    impl Attribute {
```

Comment:
@MegaBluejay why as a separate `impl Attribute` block? There is one already down bellow.